darglint2/config.py

```diff
@@ -78,23 +79,46 @@ def from_string(cls, level: str) -> "LogLevel":
             raise ValueError('Unrecognized log level, "{}"'.format(level))
 
 
-class Configuration(object):
+class Configuration:
+    """
+    A dataclass representing a configuration.
+
+    Attributes:
+        ignore: A list of error codes to ignore.
+        message_template: the template with which to format the errors.
+        style: The style of docstring.
+        strictness: The minimum strictness to allow.
+        ignore_regex: A regular expression which enables ignoring
+            functions/methods by name.
+        ignore_raise: A list of exceptions that don't need to be
+            documented.
+        ignore_properties: Bool indicating whether to ignore properties functions
+            or not.
+        enable: A list of of error codes that are disabled by default.
```

Comment:
Is this accurate? Should it say `...error codes to enable that are disabled by default`? Or perhaps `...are enabled by default`?